File: src/ERP/acme.atena.domain/DTO/Inventory/Estoque.cs
using acme.atena.domain.DTO.Person;
using acme.atena.domain.DTO.Product;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace acme.atena.domain.DTO.Inventory
{
    public  class Estoque: AbstractEntity
    {
        public Estoque()
        {
            EntradaProdutoEstoques = new HashSet<EntradaProdutoEstoque>();
            SaidaProdutoEstoques = new HashSet<SaidaProdutoEstoque>();
            EstoqueProdutos = new HashSet<EstoqueProduto>();
        }

        public string Nome { get; set; }
        public bool IsPrincipal { get; set; }
        public int EstoqueMaximo { get; set; }
        public int EstoqueMinimo { get; set; }
        public Guid EmpresaId { get; set; }
        public Empresa Empresa { get; set; }
        
        public virtual ICollection<EntradaProdutoEstoque> EntradaProdutoEstoques { get; set; }
        public virtual ICollection<SaidaProdutoEstoque> SaidaProdutoEstoques { get; set; }
        public virtual ICollection<EstoqueProduto> EstoqueProdutos { get; set; }
        
    }
}

File: site/coreui-free-angular-admin-template-master/src/app/view-model/response/inventory/estoque.ts
import { SaidaProdutoEstoque } from './saida-produto-estoque';
import { EstoqueProduto } from './estoque-produto';
import { EntradaProdutoEstoque } from './entrada-produto-estoque';
import { AbstractEntity } from "../abstract-entity";
import { Empresa } from "../person/empresa";

export class Estoque extends AbstractEntity{
  public nome: string ;
  public isPrincipal: boolean ;
  public estoqueMaximo: number ;
  public estoqueMinimo:number ;
  public empresaId: string ;
  public empresa: Empresa;

  public entradaProdutoEstoques: EntradaProdutoEstoque[] ;
  public saidaProdutoEstoques:SaidaProdutoEstoque[] ;
  public estoqueProdutos: EstoqueProduto[];
}
